src/bp/studio/media/media-router.ts

```diff
@@ -7,13 +7,16 @@ import { CustomStudioRouter } from 'studio/utils/custom-studio-router'
 
 const debugMedia = DEBUG('audit:action:media-upload')
 
+const DEFAULT_ALLOWED_MIME_TYPES = ['image/jpeg', 'image/png', 'image/gif', 'audio/mpeg', 'video/mp4']
+const DEFAULT_MAX_FILE_SIZE = '25mb'
+
 class MediaRouter extends CustomStudioRouter {
-  constructor(services: StudioServices, private botpressConfig?: BotpressConfig) {
+  constructor(services: StudioServices) {
     super('User', services)
-    this.setupRoutes()
+    void this.setupRoutes()
```

Comment:
How does this actually work ? Is the promise awaited ? or is it a floating promise ?

Reply:
Floating promise! I'll make some changes!